triggers.md

```diff
@@ -283,6 +284,13 @@ Adding `"dns-prefetch"` and `"preconnect"`, to mirror [Resource Hints](https://w
 
 Another envisioned speculative action is `"prefetch_with_subresources"`, which prefetches a document and then uses the HTML preload scanner to find other subresources that are worth preloading. Chromium currently does something similar (known as "[NoState Prefetch](https://developer.chrome.com/blog/nostate-prefetch/)") for `<link rel="prerender">`. But, we're not yet sure this feature is pulling its weight, in between the lightweight prefetch and the fully-instant prerender features, so it's not yet clear whether this will be worth integrating.
 
+### Content-Security-Policy
+
+`Speculation-Rules` is embedded as an inline declarationt within a script tag with type=”speculationrules”, and restricted by the `script-src` restriction of the `Content-Security-Policy`.
```

Comment:
```suggestion
Speculation rules can be embedded inline within a script tag with type=”speculationrules”, and restricted by the `script-src` CSP directive.
```

The HTTP header name isn't really applicable here, so we can just describe it as "speculation rules".